test/integration/stock/stock.js

```diff
@@ -7,205 +7,174 @@ const shared = require('./shared');
 describe('(/stock/) The Stock HTTP API', () => {
 
   // create new stock lots
-  it('POST /stock/lots create a new stock lots entry', () => {
-    return agent.post('/stock/lots')
-      .send(shared.movementFirstLots)
-      .then((res) => {
-        helpers.api.created(res);
-      })
-      .catch(helpers.handler);
+  it('POST /stock/lots create a new stock lots entry', async () => {
+    const res = await agent.post('/stock/lots')
+      .send(shared.movementFirstLots);
+    helpers.api.created(res);
```

Comment:
It seems like errors are not handled, i am wondering just if they can be handled

Reply:
If an error occurs the status code won't be 201

```
api.created = function created(res) {

  // make sure the response has correct HTTP headers
  expect(res).to.have.status(201);
  expect(res).to.be.json;

```